fabric-keybindings-v0/src/main/java/net/fabricmc/fabric/mixin/client/keybinding/MixinKeyBindingAccessor.java

```diff
@@ -0,0 +1,13 @@
+package net.fabricmc.fabric.mixin.client.keybinding;
+
+import net.minecraft.client.options.KeyBinding;
+import net.minecraft.client.util.InputUtil;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Accessor;
+
+@Mixin(KeyBinding.class)
+public interface MixinKeyBindingAccessor
+{
+	@Accessor("keyCode")
+	InputUtil.KeyCode fabric_getConfiguredKeyCode();
```

Comment:
Rename to `getKeyCode` and remove the String argument from the accessor annotation. Uniqueness is not a concern here.

Reply:
From a usage standpoint, does it make sense to have `getKeyCode` and `getDefaultKeyCode`? Is the intention here (`getKeyCode` being the configured one from user settings) apparent enough?

Reply:
I think it is better for it to be pure and inferred by the target's name, but if you want clarity you could use `getBoundKeyCode` (also KeyCode should be Key but that's another discussion)